### PR TITLE
Add DSEP contact info for adoption in intro.md

### PR DESCRIPTION
**Pull request by sideye**
Currently there does not appear to be any contact links or information on `intro.md`. Notably, the intro even mentions "if you’d like to modify the textbook’s content for your own course, please shoot us an email!", but there is no email address. 

This PR adds a new "Contact Us" subsection in the intro with the external adoption email address `ds-help@berkeley.edu`, along with the [data 8 instructor interest form](https://forms.gle/maRqsceCvTNnqQ3w9).

### CODE REVIEW

**Comment by choldgraf** [comment]
Hey there - it seems that a bunch of stuff in the `_build` folder were updated outside of the "intro" change. Could you revert anything that's not related to the "intro" fix you mentioned in this PR? Thanks!

**Comment by sideye** [comment]
@choldgraf it seems like all of the other updates are already implemented in `content` in master, so that when I ran `make book`, the changes went through as well. Should I just PR without running `make book`?

File: content/intro.md [hunk 1]
```diff
@@ -62,4 +62,9 @@ data analytics pedagogy. Here are a few notable resources:
   our teams coordinate technical infrastructure across classes and resources.
 * [The Jupyter for Education Handbook](https://jupyter4edu.github.io/jupyter-edu-book/)
   is a more general resource for teaching using tools in the Jupyter ecosystem, and has several
-  best-practices and tips.
\ No newline at end of file
+  best-practices and tips.
+
+## Contacting Us
+
+If you woud like to learn more about any of the tools used in Data 8 or are interested in deploying your own data 8 course,
```
Comment (choldgraf) [inline content/intro.md]:
```suggestion
If you would like to learn more about any of the tools used in Data 8 or are interested in deploying your own data 8 course, 
```

**Comment by choldgraf** [comment]
ahh gotcha - OK cool, let's merge this away :-)

**Pull request merged**